docs/content/concepts/webserver/ui.mdx

```diff
@@ -35,253 +35,603 @@ Note that several Dagster features, like schedules and sensors, require the Dags
+
+<TabGroup>
+<TabItem name="Daemons tab">
+
+#### Daemons tab
+
+- **Description**: The **Daemons** tab contains information about the [daemons](/deployment/dagster-daemon) in an Open Source Dagster deployment, including their current status and when their last heartbeat was detected.
+- **Accessed by**: Clicking the **Daemons** tab in the [**Deployment overview**](#code-locations-tab) page
+
+<!-- ![UI Deployment - Daemons tab](/images/concepts/webserver/deployment-daemons-tab.png) -->
+
+<Image
+alt="UI Deployment - Daemons tab"
+src="/images/concepts/webserver/deployment-daemons-tab.png"
+width={3060}
+height={1626}
+/>
+
+</TabItem>
+<TabItem name="Configuration tab">
+
+#### Configuration tab
+
+- **Description**: The **Configuration** tab displays information about the configuration for a Dagster deployment, which is managed through the [`dagster.yaml`](/deployment/dagster-instance) file
+- **Accessed by**: Clicking the **Configuration** tab in the [**Deployment overview**](#code-locations-tab) page
+
+<!-- ![UI Deployment - Configuration tab](/images/concepts/webserver/deployment-configuration-tab.png) -->
+
+<Image
+alt="UI Deployment - Configuration tab"
+src="/images/concepts/webserver/deployment-configuration-tab.png"
+width={3566}
+height={1942}
+/>
+
+</TabItem>
+</TabGroup>
+</TabItem>
+
+<TabItem name="Cloud">
+
+### Cloud
+
+In addition to the [**Code locations** tab](#code-locations-tab), Dagster Cloud deployments contain a few additional tabs. Click the tabs below for more information.
+
+<TabGroup>
+<TabItem name="Agents tab">
+
+#### Agents tab
+
+- **Description**: The **Agents** tab contains information about the agents in a Dagster Cloud deployment. Refer to the [Dagster Cloud agent documentation](/dagster-cloud/deployment/agents) for more info.
+- **Accessed by**: Clicking the **Agents** tab in the [**Deployment overview**](#code-locations-tab) page
+
+<!-- ![UI Cloud Deployment - Agents tab](/images/concepts/webserver/deployment-cloud-agents-tab.png) -->
+
+<Image
+alt="UI Cloud Deployment - Agents tab"
+src="/images/concepts/webserver/deployment-cloud-agents-tab.png"
+width={3060}
+height={1632}
 />
+
+</TabItem>
+<TabItem name="Environmental variables tab">
+
+#### Environment variables tab
+
+- **Description**: The **Agents** tab contains information about the environment variables configured in a Dagster Cloud deployment. Refer to the [Dagster Cloud environment variables and secrets documentation](/dagster-cloud/managing-deployments/environment-variables-and-secrets) for more info.
+- **Accessed by**: Clicking the **Environment variables** tab in the [**Deployment overview**](#code-locations-tab) page
+
+<!-- ![UI Cloud Deployment - Environment variables tab](/images/concepts/webserver/deployment-cloud-environment-variables-tab.png) -->
+
+<Image
+alt="UI Cloud Deployment - Environment variables tab"
+src="/images/concepts/webserver/deployment-cloud-environment-variables-tab.png"
```

Comment:
is it ok to have a real screenshot from our cloud deployment here? there's no secret info being displayed, but it might still be less than ideal